server/getPages.js

```diff
@@ -0,0 +1,7 @@
+import {query} from './query.js';
+
+export const getOnePage = async (req, res) => {
+  res.json(
+    await query('SELECT * FROM "pages_data" WHERE id=$1', [req.params.id])
```

Comment:
This is going to return an array with one object in it. Can we instead return just the object?